apps/angular/src/app/features/anime-table/anime-table.component.html

```diff
@@ -0,0 +1,25 @@
+<table mat-table [dataSource]="(anime$ | async) ?? []" class="anime-table">
+	@for (column of displayedColumns; track $index) {
+		<ng-container [matColumnDef]="column.key">
+			<th mat-header-cell *matHeaderCellDef>{{column.header}}</th>
+			<td mat-cell *matCellDef="let element">
+				@switch (column.key) {
+					@case ('image') {
```

Comment:
Can we use enum `ColumnKey` in `@case`? Otherwise, you can change the string in one place and forget it in another.
```suggestion
					@case (columnKey.Image ) {
```

To do this, add the field to the component:

`protected columnKey = ColumnKey;`